.env.example

```
@@ -1,3 +1,11 @@
 POSTGRES_USER=
-POSTGRES_PASSWORD=
\ No newline at end of file
+POSTGRES_PASSWORD=
+
+# AWS S3 (photo uploads)
+AWS_REGION=eu-west-2
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_S3_BUCKET_NAME=eros-photos
+AWS_CDN_BASE_URL=
+AWS_PRESIGNED_URL_TTL_MINUTES=15
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)